modules/platform/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/test/java/org/nuxeo/ecm/platform/picture/core/test/TestPictureBlobHolder.java

```diff
@@ -93,12 +88,14 @@ public void testBlobHolder() throws Exception {
         DocumentModel picturebook = session.createDocumentModel("/", "picturebook", PICTUREBOOK_TYPE_NAME);
         session.createDocument(picturebook);
         DocumentModel picture = session.createDocumentModel(picturebook.getPathAsString(), "pic1", PICTURE_TYPE_NAME);
-        picture.setPropertyValue("file:content", (Serializable) Blobs.createBlob(
-                getFileFromPath("images/exif_sample.jpg"), "image/jpeg", null, "mysample.jpg"));
+        picture.setPropertyValue("file:content",
+                (Serializable) Blobs.createBlob(getFileFromPath("images/exif_sample.jpg"), "image/jpeg", null,
+                        "mysample.jpg"));
         picture = session.createDocument(picture);
         DocumentModel picture2 = session.createDocumentModel(picturebook.getPathAsString(), "pic2", PICTURE_TYPE_NAME);
-        picture2.setPropertyValue("file:content", (Serializable) Blobs.createBlob(
-                getFileFromPath("images/exif_sample.jpg"), "image/jpeg", null, "mysample.jpg"));
+        picture2.setPropertyValue("file:content",
+                (Serializable) Blobs.createBlob(getFileFromPath("images/exif_sample.jpg"), "image/jpeg", null,
+                        "mysample.jpg"));
```

Comment:
shouldn't be in the cleanup / format commit since there is one ?

Reply:
Yes, will do that on https://github.com/nuxeo/nuxeo/pull/4260/ so :)

Reply:
Done.